sdk/iot/core/doc/mqtt_state_machine.md

```diff
@@ -136,3 +138,120 @@ _Example:_
         }
     }
 ```
+
+### Retrying Operations
+
+Retrying operations requires understanding two aspects: error evaluation (did the operation fail, should the operation be retried) and retry timing (how long to delay before retrying the operation). The IoT Embedded Devices SDK is supplying optional APIs for error classification and retry timing.

```

Comment:
>IoT Embedded Devices SDK [](start = 203, length = 24)

Naming convention on IoT Embedded Devices SDK?

Reply:
Let's discuss this in https://github.com/Azure/azure-sdk-for-c/pull/446#discussion_r391888842